lib/tests/inline-imports-test.js

```diff
@@ -391,4 +391,60 @@ fragment CustomResolutionFragment on User {
 }
     `);
   });
+
+  it('`inlineImportsWithLineToImports` relays resolveOptions when there are transitive fragments', function () {
```

Comment:
2/5 (minor preference, non-blocking)

This test confirms that the `resolveOptions` are passed through (and that seems good to do!), but it doesn't really do anything to confirm that the "real world" issue you hit works. It would be great if you could have a test that demonstrates the missing imports (that fails before the changes, and passes after).

Reply:
@rwjblue I think I should've phrased the description a little better (I just updated the description to hopefully be clearer).

We pass custom resolution strategy `resolveImport` to  `inlineImportsWithLineToImports`, and the `resolveImport` requires `resovleOptions.{pkg,pkgRoot}` to work correctly when importing a fragment from an external dependency in another fragment. So the issue in this repo is just fields on `resolveOptions` are lost when `inlineImportsWithLineToImports` is called recursively.

Reply:
That being said, I updated the test to use the same use case (importing external fragments in a fragment) as in the description.